dali/base/dadfs.cpp

```diff
@@ -9131,6 +9158,9 @@ class CInitGroups
 
     bool loadMachineMap()
     {
+        if (machinesLoaded)
```

Comment:
don't think so, CInitGroups is a utility class used and not thread safe in general and not expected to be.